### PR TITLE
Feature/login

### PR DESCRIPTION
**Pull request by srdjanstanojcic**
Added login feature with custom exception

### CODE REVIEW

File: BE/Api.Access.Layer/Controllers/UserController.cs [hunk 1]
```diff
@@ -50,11 +49,28 @@ public async Task<IActionResult> UpdateUser([FromBody] BusinessUserModel model)
 
         [ProducesResponseType((int)HttpStatusCode.OK)]
         [ProducesResponseType((int)HttpStatusCode.NotFound)]
-        [HttpGet("read")]
-        public async Task<IActionResult> GetUserByEmail([FromBody] string email)
+        [HttpGet("GetUserByEmail")]
```
Comment (srdjanstanojcic) [inline BE/Api.Access.Layer/Controllers/UserController.cs:52]: @BoskoGogic95 See if these changes of GetUserByEmail method are okay

File: BE/Business.Access.Layer/Services/Contracts/IUserService.cs [hunk 2]
```diff
@@ -1,15 +1,20 @@
 ﻿using Business.Access.Layer.Models.UserModels;
+using Data.Access.Layer.Models;
 
 namespace Business.Access.Layer.Services.Contracts
 {
     public interface IUserService
     {
         Task<Guid?> CreateUser(BusinessUserModel model);
 
-        Task<BusinessUserModel> GetUserByMail(string mail);
```
Comment (srdjanstanojcic) [inline BE/Business.Access.Layer/Services/Contracts/IUserService.cs:9]: @BoskoGogic95  Check these changes also

File: BE/Api.Access.Layer/Controllers/UserController.cs (hunk 1 above)
Comment (BoskoGogic95) [reply BE/Api.Access.Layer/Controllers/UserController.cs:52]: Yea it's okay . Thanks for notify about that changes.

File: BE/Api.Access.Layer/Controllers/UserController.cs [hunk 3]
```diff
@@ -19,9 +18,9 @@ public UserController(IUserService userService, IMapper mapper) : base(mapper)
             _userService = userService;
         }
 
-        [ProducesResponseType((int)HttpStatusCode.OK)]
-        [ProducesResponseType((int)HttpStatusCode.NotFound)]
-        [ProducesErrorResponseType(typeof(UserNotFoundException))]
+        //[ProducesResponseType((int)HttpStatusCode.OK)]
+        //[ProducesResponseType((int)HttpStatusCode.NotFound)]
+        //[ProducesErrorResponseType(typeof(UserNotFoundException))]
```
Comment (petar-kovac) [inline BE/Api.Access.Layer/Controllers/UserController.cs]: Why is this commented?

Comment (srdjanstanojcic) [reply BE/Api.Access.Layer/Controllers/UserController.cs]: resolved

**Pull request merged**